### PR TITLE
feat: language support added for es, pt, mandarin and cantonese

### LINKED ISSUE #945: feat: add support for Portuguese, Spanish, Cantonese, Mandarin
### Which at_widget package you want to add enhancement?

at_onboarding_flutter

### On what platforms do you want to add enhancement?

Android, iOS, Windows, MacOS

### Is your feature request related to a problem?

_No response_

### Describe the solution that you'd like

_No response_

### Describe alternatives that you have considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by CurtlyCritchlow**
…ntonese in at_onboarding_flutter.

**- What I did**
 closes atsign-foundation/at_client_sdk#1590 
 Updated the readme.md  and app version to version and 6.10. 
**- How I did it**

**- How to verify it**

**- Description for the changelog**

Language support added for Spanish (latin American), Portuguese (European and Brazillian) and Chinese ( mandarin and cantonese)

### CODE REVIEW

File: packages/at_onboarding_flutter/example/lib/main.dart [hunk 1]
```diff
@@ -177,16 +189,23 @@ class MyAppState extends State<MyApp> {
...
+                          items: [
+                            DropdownMenuItem(value: englishLocale, child: const Text('English')),
+                            DropdownMenuItem(value: frenchLocale, child: const Text('French')),
+                            DropdownMenuItem(value: spanishLocale, child: const Text('Spanish')),
+                            DropdownMenuItem(value: portugueseLocale, child: const Text('Portuguese')),
+                            DropdownMenuItem(
+                                value: brazillianPortugueseLocale, child: const Text('Brazilian Portuguese')),
+                            DropdownMenuItem(
+                                value: chineseTraditionalLocale, child: const Text('Chinese (Traditional)')),
+                            DropdownMenuItem(value: chineseSimplifiedLocale, child: const Text('Chinese (Simplified)')),
```
Comment (xavierchanth) [inline packages/at_onboarding_flutter/example/lib/main.dart:208]: These text items should be written in the language that they are:
i.e. `French -> Francais` (missing the accent, but you get the point)

File: packages/at_onboarding_flutter/lib/l10n/intl_zh_Hans_CH.arb [hunk 2]
```diff
@@ -0,0 +1,114 @@
+{
+    "activate_an_atSign": "激活一个@符号",
```
Comment (xavierchanth) [inline packages/at_onboarding_flutter/lib/l10n/intl_zh_Hans_CH.arb]: This doesn't look quite right, `atSign` got translated, make sure it still means "atSign" and not "at sign"

Comment (xavierchanth) [reply packages/at_onboarding_flutter/lib/l10n/intl_zh_Hans_CH.arb]: Same for other localizations

**Review by xavierchanth — CHANGES_REQUESTED** [review]
See my comments

File: packages/at_onboarding_flutter/example/lib/main.dart (hunk 1 above)
Comment (CurtlyCritchlow) [reply packages/at_onboarding_flutter/example/lib/main.dart:208]: This is in the example app, I this that would make things a bait confusing for developers looking at the example app code or running it. If you disagree I can make the changes?

Comment (xavierchanth) [reply packages/at_onboarding_flutter/example/lib/main.dart:208]: Ah, never mind then

File: packages/at_onboarding_flutter/lib/l10n/intl_zh_Hans_CH.arb (hunk 2 above)
Comment (CurtlyCritchlow) [reply packages/at_onboarding_flutter/lib/l10n/intl_zh_Hans_CH.arb]: Done, addressed it for all the languages to keep atSign as a proper noun.

**Review by xavierchanth — APPROVED** [review]

**Pull request merged**